posthog/temporal/common/worker.py

```diff
@@ -82,8 +82,12 @@ async def create_worker(
                 # given that the `duration_as_seconds` is `False`.
                 # But in Python we still need to pass floats due to type hints.
                 histogram_bucket_overrides=dict(
-                    zip(BATCH_EXPORTS_HISTOGRAM_METRICS, itertools.repeat(BATCH_EXPORTS_HISTOGRAM_BUCKETS))
-                ),
+                    zip(
+                        BATCH_EXPORTS_LATENCY_HISTOGRAM_METRICS,
+                        itertools.repeat(BATCH_EXPORTS_LATENCY_HISTOGRAM_BUCKETS),
+                    )
+                )
+                | {"batch_exports_activity_attempt": [1.0, 5.0, 10.0]},
```

Comment:
Not sure if any other buckets make sense. Anything more than 10 is probably too many, any thoughts @rossgray?